VocaDb.ResXFileCodeGenerator/StringBuilderGenerator.cs

```diff
@@ -61,39 +126,66 @@ public string Generate(Stream resxStream, GeneratorOptions options)
 		builder.Append(options.ClassName);
 		builder.AppendLine(").Assembly);");
 
-		builder.Append("        public static ");
+		builder.Append(indent);
+		builder.Append("public ");
+		builder.Append(options.StaticMembers ? "static " : "");
 		builder.Append(nameof(CultureInfo));
 		builder.Append("? ");
 		builder.Append(Constants.CultureInfoVariable);
 		builder.AppendLine(" { get; set; }");
 
 		builder.AppendLine();
 
-		static void CreateMember(StringBuilder builder, GeneratorOptions options, string name, string value)
+		static void CreateMember(string indent, StringBuilder builder, FileOptions options, string name, string value,
+			IXmlLineInfo line, HashSet<string> alreadyAddedMembers, Action<Diagnostic>? reportError)
 		{
 			string memberName;
 			bool resourceAccessByName;
 
-			if (ValidMemberNamePattern.IsMatch(name))
+			if (s_validMemberNamePattern.IsMatch(name))
 			{
 				memberName = name;
 				resourceAccessByName = true;
 			}
 			else
 			{
-				memberName = InvalidMemberNameSymbols.Replace(name, "_");
+				memberName = s_invalidMemberNameSymbols.Replace(name, "_");
 				resourceAccessByName = false;
 			}
 
-			builder.AppendLine("        /// <summary>");
+			static Location GetMemberLocation(FileOptions fileOptions, IXmlLineInfo line, string memberName) =>
+				Location.Create(fileOptions.FilePath, new(),
+					new(new(line.LineNumber - 1, line.LinePosition - 1),
+						new(line.LineNumber - 1, line.LinePosition - 1 + memberName.Length)));
```

Comment:
I would use named arguments for better readability here, but I can do it after merging this PR.